src/HTTP/Cookies/CookieJarWrapper.php

```diff
@@ -0,0 +1,125 @@
+<?php
+
+namespace ILIAS\HTTP\Cookies;
+
+use Dflydev\FigCookies\SetCookies;
+use Psr\Http\Message\ResponseInterface;
+
+/**
+ * Class CookieJarFacade
+ *
+ * Wrapper class for the FigCookies SetCookies class.
+ *
+ * @author  Nicolas Schäfli <ns@studer-raimann.ch>
+ * @package ILIAS\HTTP\Cookies
+ * @since   5.2
+ * @version 1.0.0
+ */
+class CookieJarWrapper implements CookieJar {
+
```

Comment:
@shecken: I do not agree. Same reasoning as bellow in CookieWrapper. 

However, why are constructors instead of clone used?